skyportal/tests/api/test_instrument.py

```diff
@@ -31,18 +39,25 @@ def test_token_user_post_get_instrument(super_admin_token):
             'band': 'NIR',
             'filters': ['f110w'],
             'telescope_id': telescope_id,
+            'field_data': pd.read_csv(fielddatafile)[:5].to_dict(orient='list'),
+            'field_region': Regions.read(regionsdatafile).serialize(format='ds9'),
         },
         token=super_admin_token,
     )
     assert status == 200
     assert data['status'] == 'success'
 
+    # wait for the fields to populate
+    time.sleep(15)
```

Comment:
Is there a better way to do this (like a separate query)? Adding 15 second sleep might not work on some runs of the CI.

Reply:
Well given that the (large) csv and region files are not going to be uploaded to an already running SP instance, the only real utility of `db_demo.yaml` is to add these data to a new instance. That's fine but then you don't also need to use these same large files for testing purposes. You could instead have a `Test_Fields.csv` and `Test_Regions.reg` files in the `data/` directory that have only a few lines. Then you probably wouldn't need to wait here in the test.

In addition, you could add an async await after the insertion loop in `api/instrument.py` since we're ok blocking on a POST insertion (it will happen very infrequently on a running system)

Reply:
We are just adding a few (5?) if you look in the test, it is just a short pause to be sure to be in there. The other thread still needs to start, which might take a few seconds.